joplin/base/wagtail_hooks.py

```diff
@@ -193,5 +198,32 @@ def joplin_page_listing_more_buttons(page, page_perms, is_parent=False):
             _('Delete'),
             reverse('wagtailadmin_pages:delete', args=[page.id]),
             attrs={'title': _("Delete page '{title}'").format(title=page.get_admin_display_title())},
-            priority=30
         )
+
+
+
+@hooks.register('register_rich_text_features')
+def register_help_text_feature(features):
+    """
+    Registering the `help-text` feature, which uses the `help-text` Draft.js block type,
+    and is stored as HTML with a `<div class="help-text">` tag.
+    """
+    feature_name = 'rich-text-button-link'
+    type_ = 'rich-text-button-link'
+
+    control = {
+        'type': type_,
+        'label': 'Button',
+        'description': 'Make me look like a button',
+        # Optionally, we can tell Draftail what element to use when displaying those blocks in the editor.
+        'element': 'div',
+    }
+
+    features.register_editor_plugin(
+        'draftail', feature_name, draftail_features.BlockFeature(control)
+    )
+
+    features.register_converter_rule('contentstate', feature_name, {
+        'from_database_format': {'div.rich-text-button-link': BlockElementHandler(type_)},
+        'to_database_format': {'block_map': {type_: {'element': 'div', 'props': {'class': 'usa-button-primary rich-text-button-link'}}}},
```

Comment:
I think MVP requirements are button-primary only, do we know if we'll need to do secondary in the future? If so it'd be cool to make an issue for it.

Reply:
IMO I think this is already identified as a pretty edge case (only one page that even needs it so far)

The path is clear and I think the code is obvious enough that we can revisit in the future, but we could add an additional code comment as a note to future self